.env.example

```diff
@@ -23,11 +28,10 @@ TRACECAT__API_URL=http://api:8000
 # to start ngrok and update this with the forwarding URL
 TRACECAT__PUBLIC_RUNNER_URL=http://localhost:8000
```

Comment:
does this need to change? this is used to construct the public facing webhook url (it should also have a more representive name)

Reply:
Can be changed to localhost/api/